3_javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,234 @@
+//Item array removal
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+ 
+  
+  // Write some code here
+  
+  const nameToRemove = names.splice(1,1);
+  // Code done
+  
+  console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+//When will we be there??
+function calculateTravelTime(travelInformation) {
+    const { speed, destinationDistance } = travelInformation;
+  
+    // Calculate the total time in hours
+    const totalHours = destinationDistance / speed;
+  
+    // Convert totalHours to separate hours and minutes
+    const hours = Math.floor(totalHours);
+    const minutes = Math.round((totalHours - hours) * 60);
+  
+    // Format the result
+    const formattedTime = `${hours} hours and ${minutes} minutes`;
+  
+    return formattedTime;
+  }
+  
+  const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  
+  const travelTime = calculateTravelTime(travelInformation);
+  console.log(travelTime); // Output: 8 hours and 38 minutes
+  
```

Comment:
Great job on de-structuring the travelInformation object. This makes the code cleaner and more readable.
Your calculations for the total time in hours and then converting them to hours and minutes are accurate and correctly implemented.
The code is well-structured and the variable names are descriptive, making it easy to follow.